### PR TITLE
doc: Fix typo and grammar in doc files

### PR DESCRIPTION
**Pull request by henrychoi7**
Hi, I just found some typo and grammar issues.
Also, some lines of the document should be ignored markdown formatting, so I fixed it by using `\`.

Signed-off-by: Handong Choi <wschd770@gmail.com>

### CODE REVIEW

File: doc/uftrace-live.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ COMMON OPTIONS
 
 \--no-libcall
 :   Do not record library function invocations.  Library calls are normally
-    traced by hooking the dynamic linker's resolve function in the PLT.
+    traced by hooking the dynamic linker to resolve function in the PLT.
```
Comment (namhyung) [inline doc/uftrace-live.md]: Hmm.. I'd rather say "hooking calls to the resolver function of dynamic linker in the PLT".

File: doc/uftrace-record.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ COMMON OPTIONS
 
 \--no-libcall
 :   Do not record library function invocations.  Library calls are normally
-    traced by hooking the dynamic linker's resolve function in the PLT.
+    traced by hooking the dynamic linker to resolve function in the PLT.
```
Comment (namhyung) [inline doc/uftrace-record.md]: Ditto.

File: doc/uftrace-dump.md [hunk 3]
```diff
@@ -107,7 +107,7 @@ COMMON ANALYSIS OPTIONS
 
 -r *RANGE*, \--time-range=*RANGE*
 :   Only show functions executed within the time RANGE.  The RANGE can be
-    \<start\>~\<stop\> (separated by "~") and one of \<start\> and \<stop\> can
+    \<start\>\~\<stop\> (separated by "\~") and one of \<start\> and \<stop\> can
```
[the remark this answers is not in the document]
Comment (honggyukim) [reply doc/uftrace-dump.md]: It seems that there's no difference having `\` or not.

[the remark this answers is not in the document]
Comment (henrychoi7) [reply doc/uftrace-dump.md]: @namhyung I checked it, and it turned out to be okay.

image

File: doc/uftrace-live.md (hunk 1 above)
Comment (henrychoi7) [reply doc/uftrace-live.md]: @namhyung Thank you for your opinion. It's more likely natural 👍Fixed it right away.

File: doc/uftrace-record.md (hunk 2 above)
Comment (henrychoi7) [reply doc/uftrace-record.md]: @namhyung Fixed 😄

File: doc/uftrace-dump.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (honggyukim) [reply doc/uftrace-dump.md]: Hmm.. It seems that it doesn't work on your environment. If so, I'm okay with this change.

File: doc/uftrace-live.md (hunk 1 above)
Comment (henrychoi7) [reply doc/uftrace-live.md]: Working done.

File: doc/uftrace-record.md (hunk 2 above)
Comment (henrychoi7) [reply doc/uftrace-record.md]: Working done.

**Comment by henrychoi7** [comment]
@namhyung Sorry, I forgot to push the commit.
Commit updated.

File: doc/uftrace-live.md [hunk 4]
```diff
@@ -254,8 +254,8 @@ filter; an opt-in filter with `-F`/`--filter` and an opt-out filter with
 replay time.
 
 The first one is an opt-in filter. By default, it doesn't trace anything.  But
-when one of the specified functions is executed, tracing is started.  When the
-function returns, tracing is stopped again.
+when one of the specified functions is executed, tracing will be started.  When the
+function returns, tracing will be stopped again.
```
Comment (honggyukim) [inline doc/uftrace-live.md]: It seems that it doesn't have to be changed. Please correct me if I'm wrong.

File: doc/uftrace-live.md [hunk 5]
```diff
@@ -307,7 +307,7 @@ but only `b()` and `c()`.
 
 The second type of filter is opt-out. By default, everything is traced, but when
 one of the specified functions is executed, tracing stops.  When the excluded
-function returns, tracing is started again.
+function returns, tracing will be started again.
```
Comment (honggyukim) [inline doc/uftrace-live.md]: Ditto.

File: doc/uftrace-record.md [hunk 6]
```diff
@@ -205,8 +205,8 @@ filter; an opt-in filter with `-F`/`--filter` and an opt-out filter with
 replay time.
 
 The first one is an opt-in filter. By default, it doesn't trace anything.  But
-when one of the specified functions is executed, tracing is started.  When the
-function returns, tracing is stopped again.
+when one of the specified functions is executed, tracing will be started.  When the
+function returns, tracing will be stopped again.
```
Comment (honggyukim) [inline doc/uftrace-record.md]: Ditto.

File: doc/uftrace-record.md [hunk 7]
```diff
@@ -256,7 +256,7 @@ but only `b()` and `c()`.
 
 The second type of filter is opt-out. By default, everything is traced, but when
 one of the specified functions is executed, tracing stops.  When the excluded
-function returns, tracing is started again.
+function returns, tracing will be started again.
```
Comment (honggyukim) [inline doc/uftrace-record.md]: Ditto.

File: doc/uftrace-replay.md [hunk 8]
```diff
@@ -128,8 +128,8 @@ is called it receives two types of function filter; an opt-in filter with
 be applied either at record time or replay time.
 
 The first one is an opt-in filter. By default, it doesn't show anything.
-But when one of the specified functions is met, printing is started.
-When the function returns, printing is stopped again.
+But when one of the specified functions is met, printing will be started.
+When the function returns, printing will be stopped again.
```
Comment (honggyukim) [inline doc/uftrace-replay.md]: Ditto.

File: doc/uftrace-live.md (hunk 4 above)
Comment (henrychoi7) [reply doc/uftrace-live.md]: I agree with you. Restore fix done.

File: doc/uftrace-live.md (hunk 5 above)
Comment (henrychoi7) [reply doc/uftrace-live.md]: Fix done.

File: doc/uftrace-record.md (hunk 6 above)
Comment (henrychoi7) [reply doc/uftrace-record.md]: Fix done.

File: doc/uftrace-record.md (hunk 7 above)
Comment (henrychoi7) [reply doc/uftrace-record.md]: Fix done.

File: doc/uftrace-replay.md (hunk 8 above)
Comment (henrychoi7) [reply doc/uftrace-replay.md]: Fix done.

File: doc/uftrace-replay.md [hunk 9]
```diff
@@ -179,7 +179,7 @@ not at record time.
 
 The second type of filter is opt-out. When used, everything is shown by default,
 but printing stops once one of the specified functions is met.  When the given
-function returns, printing is started again.
+function returns, printing will be started again.
```
Comment (honggyukim) [inline doc/uftrace-replay.md]: Here is one more.

File: doc/uftrace-record.md [hunk 10]
```diff
@@ -710,7 +710,7 @@ Please see `uftrace-script`(1) for details about scripting.
 WATCH POINT
 ===========
 The uftrace watch point is to display certain value only if it's changed.
-It's conceptually same as debugger's but only works at function entry and exit
+It's conceptually same as the debugger but only works at function entry and exit
```
Comment (honggyukim) [inline doc/uftrace-record.md]: I don't know if `the` is needed here.

File: doc/uftrace-replay.md (hunk 9 above)
Comment (henrychoi7) [reply doc/uftrace-replay.md]: Fix done.

File: doc/uftrace-record.md (hunk 10 above)
Comment (henrychoi7) [reply doc/uftrace-record.md]: Fixed to `other debuggers`. done.

File: doc/uftrace-dump.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (namhyung) [reply doc/uftrace-dump.md]: Please remove this part.

[the remark this answers is not in the document]
Comment (namhyung) [reply doc/uftrace-dump.md]: Please remove this part, rest looks good

[the remark this answers is not in the document]
Comment (henrychoi7) [reply doc/uftrace-dump.md]: @namhyung Oh, I thought the opposite way. If "~" sign is needed, then I'll remove the escape letters including in other files as well. Thank you for the review! 🙂

**Comment by henrychoi7** [comment]
Removed the escape letters before `~` to show it in man page.

**Pull request merged**